talkative_robot.rb

```diff
@@ -0,0 +1,238 @@
+require 'pry'
+
+
+  
+people = []  # replace when done testing
+
+
+###  remove this block when done testing -- start here
+# author = { :name=>"Martin", :age=>10, :gender=>"M", :shopping_cart=>[], :author=> "yes", :description=>"boy", :initial=>"M", :destination=>"n" }
+# user   = { :name=>"Ernest", :age=>10, :gender=>"M", :shopping_cart=>[], :grocery_list => ["apples", "oranges", "peaches", "kale"], :author=> nil, :description=>"boy", :initial=>"M", :destination=>"n" }
+# people = [author, user]
+###  remove this block when done testing -- end here
+
+
+
+def number_of_linefeeds(num)
+  num.times {puts}
+end
+
+def print_is_user_ernest_message(user)
+  puts user[:name] == "Ernest" ? "I knew it was you!" : "Sorry, you're not Ernest."
+end
+
+
+def get_user_name
+	puts "What's your name?"
+ 	gets.chomp.capitalize
```

Comment:
Tabbing looks weird. May just be GitHub. If it looks right on your screen, ignore this comment.